### PR TITLE
Add generated Batch AI service

### PR DESCRIPTION
**Pull request by RikkiGibson**
Adds generated code for the Batch AI service.

Gulpfile changes, specifically removing the Generated/ path component, depends on autorest.csharp changes in https://github.com/Azure/autorest.csharp/pull/35.

### CODE REVIEW

File: src/ResourceManagement/BatchAI/Microsoft.Azure.Management.BatchAI.Fluent.csproj [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <PackageId>Microsoft.Azure.Management.BatchAI.Fluent</PackageId>
+        <Version>1.3.1</Version>
```
Comment (hovsepm) [inline src/ResourceManagement/BatchAI/Microsoft.Azure.Management.BatchAI.Fluent.csproj]: >1.3.1 [](start = 17, length = 5)

please change this to 1.3.0 - we do the version bump before release for all the packages

**Comment by hovsepm** [comment]
@RikkiGibson will this library be shipped in Java as well? I'm not able to find a PR to Java.

File: src/ResourceManagement/BatchAI/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+// </auto-generated>
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Azure BatchAI Management Library")]
+[assembly: AssemblyDescription("Provides Microsoft Azure BatchAI management APIs.")]
+
+[assembly: AssemblyVersion("1.0.0.60")]
+[assembly: AssemblyFileVersion("1.3.1.0")]
```
Comment (hovsepm) [inline src/ResourceManagement/BatchAI/Properties/AssemblyInfo.cs]: >1.3.1.0 [](start = 32, length = 7)

Same comment - please change the version to 1.3.0 (I assume we are not going to publish this package only, right?)

File: src/ResourceManagement/BatchAI/BatchAIManager.cs [hunk 3]
```diff
@@ -0,0 +1,95 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// </auto-generated>
```
Comment (hovsepm) [inline src/ResourceManagement/BatchAI/BatchAIManager.cs]: nit: Managers are not autogenerated. please remove the "<auto-generated>" tag.

File: src/ResourceManagement/BatchAI/Microsoft.Azure.Management.BatchAI.Fluent.csproj [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        <AssemblyOriginatorKeyFile>../../../tools/MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
+    </PropertyGroup>
+    <PropertyGroup>
+        <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.3' ">$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+        <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard1.3' ">1.6.1</NetStandardImplicitPackageVersion>
+        <TargetFrameworks>net452;netstandard1.3</TargetFrameworks>
+    </PropertyGroup>
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.10" />
+        <PackageReference Include="Microsoft.Azure.Management.ResourceManager.Fluent" Version="1.3.1" />
```
Comment (hovsepm) [inline src/ResourceManagement/BatchAI/Microsoft.Azure.Management.BatchAI.Fluent.csproj]: >1.3.1 [](start = 95, length = 5)

there is no such version - please change this to 1.3.0

**Comment by RikkiGibson** [comment]
Did I just overlook the local build errors or should I be doing something besides “msbuild Fluent.SDK.sln”?

File: src/ResourceManagement/BatchAI/Microsoft.Azure.Management.BatchAI.Fluent.csproj (hunk 1 above)
Comment (RikkiGibson) [reply src/ResourceManagement/BatchAI/Microsoft.Azure.Management.BatchAI.Fluent.csproj]: On it. I had just copied the version number being used in Java.

**Comment by RikkiGibson** [comment]
The Java version of this is ready but needs to be migrated to azure-libraries-for-Java.

**Comment by hovsepm** [comment]
@RikkiGibson there is a PS script that is executed on CI before build process that checks for Sync Methods and License header in each file. It is not a part of ms build.

**Comment by RikkiGibson** [comment]
I think I fixed everything mentioned in review comments.

**Pull request merged**